packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/tests/index.test.js

```diff
@@ -81,12 +72,25 @@ const makeApp = (history) => {
   );
 };
 
+function getElementByName(container, elementType, name) {
+  return container.querySelector(`${elementType}[name="${name}"]`);
+}
```

Comment:
I think you'd better off using `.getByRole()` in these cases.

Reply:
unfortunately no, because I use this function with elements that have not the role attribut

Reply:
But you're trying to find a button? So you findByRole passing the expected label as the second arg – https://testing-library.com/docs/queries/byrole/#api

Reply:
not just a button, I use the same function for a textarea and an input element

Reply:
All of which have ARIA roles?

Reply:
I have removed completely the test of the submission because I have no idea how to trigger the redirect after the submission, let me know if it is ok now please